Source/WebCore/platform/ProcessIdentity.h

```diff
@@ -52,6 +52,8 @@ class ProcessIdentity {
     // Returns true for a process identity or false on empty identity.
     WEBCORE_EXPORT operator bool() const;
 
+    WEBCORE_EXPORT ProcessIdentity clone() const;
```

Comment:
The copy pattern should preferably be uniform.
MachSendRight and ProcessIdentity could have the same name.
copy(), clone(), duplicate(), but somehow uniformly same name.